docs/contribute/index.md

```diff
@@ -6,5 +6,41 @@ nav_order: 4
 
 # How to contribute
 
+Do you want to help maintaining and extending this website? 
+
+{: .note }
+Please note: we work with GitHub to maintain en extent the content of the knowledge base. Creating a GitHub account is free. If you want to contribute please choose Sign Up on the [GitHub homepage](https://github.com).
+
+## Write or audit content for the knowledge base
+
+From August 22, '25 we will publish topics to be written for the knowledge base [labeled documentation](https://github.com/wpaccessibility/wp-a11y-docs/issues?q=is%3Aissue%20state%3Aopen%20label%3Adocumentation) in our GutHub repository.
```

Comment:
"Starting August 22nd" would be a more typical english phrase.

Fix "Guthub" => "Github" at end of sentence.